packages/amplication-client/src/Application/code-view/StorageBaseAxios.ts

```diff
@@ -0,0 +1,75 @@
+import axios, { AxiosInstance } from "axios";
+import { getToken } from "../../authentication/authentication";
+import { REACT_APP_STORAGE_GATEWAY_URL } from "../../env";
+import { NodeTypeEnum } from "./NodeTypeEnum";
+import assert from "assert";
+
+export type StorageResponseType = {
+  count: number;
+  result: StorageResponseFile[];
+  success: true;
+};
+
+export type StorageResponseFile = {
+  name: string;
+  type: NodeTypeEnum;
+  path: string;
+};
+
+export class StorageBaseAxios {
```

Comment:
I think we can upgrade this wrapper interceptors for retry, cancel, on load, error etc...